pkg/minikube/kubeconfig/config.go

```diff
@@ -178,3 +182,32 @@ func decode(data []byte) (*api.Config, error) {
 
 	return config.(*api.Config), nil
 }
+
+// GetKubeConfigStatus verifys the ip stored in kubeconfig.
+// If incorrect, the ip is overwritten.
+func GetKubeConfigStatus(ip string, filename string) (string, error) {
+	// if ip == "" {
+	// 	return "", fmt.Errorf("Error, empty ip passed")
+	// }
+	ks := state.None.String()
+	kip := state.None.String()
+	con, err := ReadConfigOrNew(filename)
+	if err != nil {
+		return "", err
+		// return "", errors.Wrap(err, "Error getting kubeconfig status")
+	}
+	kip = con.Clusters["minikube"].Server
+
+	if strings.Contains(kip, ip) {
+		ks = "Correctly configured"
+	} else {
+		con.Clusters["minikube"].Server = "https://" + ip + ":" + strconv.Itoa(constants.APIServerPort)
+		err = WriteConfig(con, filename)
+		if err != nil {
+			ks = "VM IP incorrectly configured, unable to reconfigure"
+		} else {
+			ks = "VM IP incorrectly configured, reconfigured"
```

Comment:
This should probably be split into a separate command and status tells the user to run the additional command.  Something like `minikube update-context`  this command could even be used to update AND set the kubectl context to use minikube in case people are using multiple contexts.